plugin/storage/integration/cassandra_test.go

```diff
@@ -79,10 +79,7 @@ func (s *CassandraStorageIntegration) initializeCassandra(t *testing.T) {
 	spanReader, err := f.CreateSpanReader()
 	require.NoError(t, err)
 	s.TraceReader = v1adapter.NewTraceReader(spanReader)
-	s.ArchiveSpanReader, err = f.CreateArchiveSpanReader()
-	require.NoError(t, err)
-	s.ArchiveSpanWriter, err = f.CreateArchiveSpanWriter()
-	require.NoError(t, err)
+	s.ArchiveTraceReader, s.ArchiveTraceWriter = v1adapter.InitializeArchiveStorage(s.factory, zaptest.NewLogger(t, zaptest.WrapOptions(zap.AddCaller())))
```

Comment:
this is not equivalent. InitializeArchiveStorage never fails, but we want to detect errors if archive objects cannot be created.